tokio/src/net/unix/datagram/socket.rs

```diff
@@ -507,9 +508,12 @@ impl UnixDatagram {
     /// # }
     /// ```
     pub async fn recv_from(&self, buf: &mut [u8]) -> io::Result<(usize, SocketAddr)> {
-        self.io
-            .async_io(mio::Ready::readable(), |sock| sock.recv_from(buf))
-            .await
+        let (n, addr) = self
+            .io
+            .async_io(mio::Interest::WRITABLE, |sock| sock.recv_from(buf))
```

Comment:
`recv_from` -> `WRITABLE` 🤔

Reply:
Good call. There is no test for this apparently.